ffighter

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# changing directory to be able to use relative paths.
+starting_working_directory=$(pwd)
+rootDir="/home/open-schnick/Documents/projects/filefighter/ClientSetup/" # how do we get that?
```

Comment:
https://www.cyberciti.biz/faq/linux-unix-shell-script-find-out-in-which-directory-script-file-resides/

Reply:
https://www.golinuxcloud.com/get-script-name-get-script-path-shell-script/

Easier tutorial